### ISSUE TITLE
Upgrade issues: Installer fails to start SW. Hotkey not functional after starting manually

### ISSUE THREAD

**Issue by mzso** [Belphemur/SoundSwitch#447]
So far I upgraded twice. And this happened both times. Not much to add to SW not starting.
The hotkey fails to function until a system reboot. Or if I rebind the hotkey to something else and then back to the original. Both are cumbersome workarounds.

**Comment by Belphemur** [comment]
Hello mzso,

Which version of SoundSwitch ?

Can you try to do a clean re-install ?

**Comment by mzso** [comment]
[**@Belphemur**](https://github.com/Belphemur) commented on [2020. máj. 17. 19:25 CEST](https://github.com/Belphemur/SoundSwitch/issues/447#issuecomment-629832214 "2020-05-17T17:25:27Z - Replied by Github Reply Comments"):
> Hello mzso,
> 
> Which version of SoundSwitch ?

The last two. So now I'm on 5.1

**Comment by Belphemur** [comment]
Can you uninstall the beta 5.1


You shouldn't have issue with the 5.0.4.


The beta is unstable.


Sent from Nine
________________________________

**Comment by mzso** [comment]
It's exactly the same. Except it also fails to close the running SW instance.
And when I tried to double click on it it crashed twice.

**Comment by mzso** [comment]
And it keeps harassing me with the beta version download even though I just disabled the option to look for beta versions.

Furthermore it shows to be 5.1.0.32818, even though I ran the "SoundSwitch_v5.0.4.31153_Release_Installer.exe" installer.

**Comment by mzso** [comment]
Tried installing with the remove settings option, and now it freezes instantly. When I try to open it...

**Comment by Belphemur** [comment]
1. Uninstall the version you're using
  1. Remove all data (the uninstaller will ask if you want to do this)
1. Download 5.0.4
  1. Install it

It should resolve your issue.

**Comment by mzso** [comment]
I experienced much of the same issues with stable. Upgrading to 5.3.0 seems to have went fine. (Though annoyingly .net was required to be upgraded)
But then I got the notification again, so I tried that as well. The installer failed to start SW, functionality broke, it crashed when I clicked the icon. And it wasn't successfully upgraded. But this time at least a restart fixed it. I could run the installer again and this time it succeeded.
